src/System.Console/ref/System.Console.cs

```diff
@@ -24,6 +25,7 @@ public static partial class Console
         public static void SetError(System.IO.TextWriter newError) { }
         public static void SetIn(System.IO.TextReader newIn) { }
         public static void SetOut(System.IO.TextWriter newOut) { }
+        public static bool WindowWidth { get { return default(bool); } set { } }
```

Comment:
Looks like we need to get our ApiCompat tooling working to catch these type of errors.